src/includes/only-error-issues-note.mdx

```diff
@@ -0,0 +1,5 @@
+<Note>
+
+This feature is only applicable to error issues. Other categories of issues (e.g., Performance Issues) do not support this feature.
```

Comment:
Oh making this an include is a good idea. I'll adopt this for my PR when I get back! I suggest the following edit:
```suggestion
This feature is only applicable for [error issues](/product/issues/#error-issues/). Other categories of issues (e.g., Performance Issues) do not support this feature.
```

Reply:
Makes sense, thanks!